leakage-test/leakage-test_shacl.ttl

```diff
@@ -0,0 +1,145 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix leakage-test: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/leakage-test> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+leakage-test:LeakageTestShape
```

Comment:
I am missing the GeneralShape (see readme).
If it does not apply to your use case feel free to raise an issue and assign it to the  functional CODEOWNER (not me :))

Reply:
You also need to add `general` to the instance file.
Interesting why the schema validation passes in this case because you defined `GeneralShape` as mandatory. In my point of view the validation should have failed. 
Seems to be a bug in the `pyshacl` lib :(